src/index.html

```diff
@@ -7,7 +7,50 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="container">
```

Comment:
do you need this `container` with one style `display: block`? it's block by default, so you don't need to specify it one more time
and this container is redundant, you don't use it